src/pykeen/models/base.py

```diff
@@ -580,6 +622,9 @@ def __init_subclass__(cls, autoreset: bool = True, **kwargs):  # noqa:D105
         if autoreset:
             _add_post_reset_parameters(cls)
 
+    def _get_entity_len(self, mode: Mode = None) -> int:  # noqa:D105
+        return self.num_entities
```

Comment:
this assumes that mode is `transductive`?

Reply:
Yes, correct

Reply:
Again, we might be able to do more abstraction later

Reply:
Made more explicit in c9ac636c